### PR TITLE
[Website] Ft loading page

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)
- Features the loading component.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Fetch and checkout the _ft-loading-page_ branch

#### What are the relevant tickets?
- [WEB-253]

#### Screenshots
Screenshot from 2022-07-12 13-04-23
 

[WEB-253]: https://airqoteam.atlassian.net/browse/WEB-253?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/Pipfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+django-filter = "*"
+djangorestframework = "*"
+dj-database-url = "*"
+drf-yasg = "*"
+invoke = "*"
+python-decouple = "*"
+psycopg2-binary = "*"
+django-storages = {version = "*", extras = ["google"]}
+
+[requires]
```
Comment (ja-odur) [inline website/Pipfile:20]: @FaithDaka Let's remove this file out. We dont want to cause conflict with maintaining deployments. You can keep it locally in your environment incase you are using pipenv.

File: website/frontend/App.js [hunk 2]
```diff
@@ -1,47 +1,58 @@
...
+const ResearchPage = React.lazy(() => import('src/pages/ResearchPage'));
+const CommunityPage = React.lazy(() => import('src/pages/CommunityPage'));
+const AfricanCitiesPage = React.lazy(() => import('src/pages/AfricanCitiesPage'));
+const AboutUsPage = React.lazy(() => import('src/pages/AboutUsPage'));
+const ContactUsPage = React.lazy(() => import('src/pages/ContactUs/ContactUs'));
+const ContactForm = React.lazy(() => import('src/pages/ContactUs/ContactForm'));
+const GetInvolved = React.lazy(() => import('src/pages/GetInvolved'));
+const Register = React.lazy(() => import('src/pages/GetInvolved/Register'));
+const CheckMail = React.lazy(() => import('src/pages/GetInvolved/CheckMail'));
+const Feedback = React.lazy(() => import('src/pages/ContactUs/Feedback'));
```
Comment (ja-odur) [inline website/frontend/App.js:19]: Neat!

**Review by ja-odur — CHANGES_REQUESTED** [review]

File: website/Pipfile (hunk 1 above)
Comment (FaithDaka) [reply website/Pipfile:20]: I had added it to the .gitignore file in highlights branch. Let me pull changes and merge.

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**